week_1_2.py

```diff
@@ -0,0 +1,100 @@
+"""
+
+TASK 2
+
+Write a base class to represent a student. Below is a starter code.
+Feel free to add any more new features to your class.
+As a minimum a student has a name and age and a unique ID.
+
+Create a new subclass from student to represent a concrete student doing a specialization, for example:
+Software Student and Data Science student.
+
+"""
```

Comment:
10/10 Really nice!

A few specific thoughts on convention and structure but answers all the requirements nicely.